src/ExpressionEvaluator/Core/Source/ExpressionCompiler/ExpressionCompiler.csproj

```diff
@@ -13,6 +13,7 @@
     <TargetFrameworkIdentifier>.NETPortable</TargetFrameworkIdentifier>
     <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <DefineConstants>$(DefineConstants);EXPRESSIONCOMPILER</DefineConstants>
```

Comment:
We were already doing this for the majority of <DefineConstants> definitions elsewhere.